### PR TITLE
Disable zoom on scroll wheel on map

### PR DESCRIPTION
**Pull request by jaimerson**

### CODE REVIEW

**Comment by phillipp-oliveira** [comment]
:shipit:

**Comment by hudsonmarinho** [comment]
:shipit:

File: app/assets/javascripts/map.js [hunk 1]
```diff
@@ -6,6 +6,8 @@ $(function() {
     L.tileLayer('https://{s}.tiles.mapbox.com/v3/mpivaa.kgcn043g/{z}/{x}/{y}.png',
       { attribution: '<a href="http://www.mapbox.com/about/maps/" target="_blank">Terms &amp; Feedback</a>' })
       .addTo(this.map);
+    this.map.doubleClickZoom.disable();
```
Comment (mpivaa) [inline app/assets/javascripts/map.js]: Doble click deixa

**Comment by hudsonmarinho** [comment]
:shipit:

**Pull request merged**